coralproject/talk
# Use <button> for dropdown to fix IOS bug

## What does this PR do?
- https://www.pivotaltracker.com/story/show/157635036

## How do I test this PR?

- Use iPhone in `portrait` mode or preferably iPad and use the `Digest Frequency DropDown`. 
- There should be no weird jumping and scrolling
